_topics/en/coinjoin.md

```diff
@@ -84,10 +91,33 @@ see_also:
     link: topic payjoin
   - title: "Coinjoin (Bitcoin Wiki: Privacy)"
     link: https://en.bitcoin.it/Privacy#CoinJoin
+  - title: "TumbleBit: An Untrusted Bitcoin-Compatible Anonymous Payment Hub"
```

Comment:
Seems like this might be a primary source?

Reply:
It is a standalone research paper, so imo yes.